src/unix/bsd/netbsdlike/openbsd/x86_64.rs

```diff
@@ -3,6 +3,140 @@ use PT_FIRSTMACH;
 pub type c_long = i64;
 pub type c_ulong = u64;
 pub type c_char = i8;
+pub type ucontext_t = sigcontext;
+
+s_no_extra_traits! {
+    #[repr(C, packed)]
+    pub struct __c_anonymous_fxsave64 {
```

Comment:
the struct isn't anonymous, it is called `struct fxsave64` in OpenBSD code source.

if you need to not not test it because it is failing, please just explicitly not test it (and comment why in `test_openbsd()`)